modules/calib3d/CMakeLists.txt

```diff
@@ -9,4 +9,4 @@ endif()
 ocv_define_module(calib3d opencv_imgproc opencv_features2d opencv_flann ${debug_modules}
     WRAP java objc python js
 )
-ocv_target_link_libraries(${the_module} ${LAPACK_LIBRARIES})
+ocv_target_link_libraries(${the_module} ${LAPACK_LIBRARIES} ${CMAKE_SHARED_LINKER_FLAGS})
```

Comment:
This is wrong change:
- shared libraries build should already have these flags automatically
- static libraries build goes broken due to mess with shared flags.